app/api/tickets.py

```diff
@@ -59,6 +66,7 @@ def before_create_object(self, data, view_kwargs):
             if not event.is_payment_enabled():
                 raise UnprocessableEntity(
                     {'event_id': data['event']}, "Event having paid ticket must have a payment method")
+        validate_ticket_price(data.get('type'), data['price'])
```

Comment:
You need to pass call this function only if `data.get('price')` is not `None` @mrsaicharan1